01-filedir.md

```diff
@@ -432,7 +432,7 @@ Run `pwd` and `ls -F` to ensure that we're in the directory we expect.
 > mean "the current user's home directory". For example, if Nelle's home
 > directory is `/Users/nelle`, then `~/data` is equivalent to
 > `/Users/nelle/data`. This only works if it is the first character in the
-> path: `here/there/~/elsewhere` is *not* `/Users/nelle/elsewhere`. 
+> path: `here/there/~/elsewhere` is *not* `here/there//Users/nelle/elsewhere`. 
```

Comment:
Yes - but I think it's OK to be a little sloppy here as we really just want to stress on the idea. We'd have to add a side note that `//` is a valid separator, but that distracts from the main point that `~` doesn't expand to anything in the middle of a path.